Directory.Build.props

```diff
@@ -222,10 +222,16 @@
     <LicenseUrl>https://github.com/dotnet/$(GitHubRepositoryName)/blob/master/LICENSE.TXT</LicenseUrl>
     <RuntimeIdGraphDefinitionFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'pkg', 'runtime.json'))</RuntimeIdGraphDefinitionFile>
     <LicenseFile>$(MSBuildThisFileDirectory)LICENSE.TXT</LicenseFile>
+    <PackageProjectUrl>$(ProjectUrl)</PackageProjectUrl>
+    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Copyright>$(CopyrightNetFoundation)</Copyright>
     <PackageThirdPartyNoticesFile>$(MSBuildThisFileDirectory)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <ReleaseNotes>https://go.microsoft.com/fwlink/?LinkID=799421</ReleaseNotes>
+    <PackageReleaseNotes>$(ReleaseNotes)</PackageReleaseNotes> 
+    <Title>$(MSBuildProjectName)</Title>
```

Comment:
That property should be unnecessary as Title already defaults to PackageId which defaults to the project's name.

Reply:
The title is not getting set without this.

Reply:
You can just remove that property at all. If Title isn't provided, the PackageId which defaults to the project's name will be used. If a package wants to specify a title, they can do so in in the csproj directly.

Reply:
@ericstj thoughts? I would just let NuGet default to PackageId and only specify a title for packages that explicitly want to.

Reply:
We should avoid customizations unless they are absolutely necessary.  If we are somehow breaking the default behavior we should fix that rather than duplicate the defaults.

Reply:
![image](https://user-images.githubusercontent.com/7412651/112177573-5a1e8800-8bf9-11eb-8287-b3855b497c2d.png)

Right. So my suggestion is to remove L233.